terragrunt.hcl

```diff
@@ -117,6 +125,7 @@ inputs = {
   location = get_env("LOCATION")
   naming_suffix = get_env("NAMING_SUFFIX")
   truncated_naming_suffix = get_env("TRUNCATED_NAMING_SUFFIX")
+  environment = get_env("ENVIRONMENT")
```

Comment:
I guess this maybe comes into the apps deploy on CI story but I imagine tf will be unhappy with deploying a repo that already exists that doesn't exist in the unshared state between environments. Wonder if it's easier to have a `count` on the resource(s) that are shared across environments now (i.e. if exists count = 0)?